### ISSUE TITLE
Raw data problem

### ISSUE THREAD

**Issue by h19920918** [jayleicn/TVQA#9]
Hi, I am a person who is working on your dataset.

I have a problem while using your raw image data.
The function 'interval2frame' located in 'preprocessing.py' returns incorrect range among images in a video.
It means that 'located_frame' indicates not existing image by a ts.

According to what I found,
    set_name = Validation
    index = 12342 in len(data_dicts)
    raw_image_folder_name = 'met_s04e02_seg02_clip17'
    total number of frames in the folder = 197
    Extracted range from the folder = [209, 279]

I found this problem in the training set = 3 and validation set = 1.
There is no instance which has this problem in the test set.

Please give to me your advise.

**Comment by jayleicn** [comment]
Hi, You can simply drop the corrupted ones for now, since they only account for < 0.01% of the data. I'll check and try to fix this issue recently.

**Comment by aurooj** [comment]
I had a similar problem, where located_ts ends are more than the video frame counts. 

For instance, a located_ts would be [start, 177] but video frame lengths are 174 let's say.

In such cases, I picked the minimum of video_frame_length and tstamp.end
Do you think that should be fine or you suggest dropping those as well?

**Comment by jayleicn** [comment]
I think it should be fine as I did the same.
